config/common/group_vars/k8s-cluster/ck8s-k8s-cluster.yaml

```diff
@@ -0,0 +1,89 @@
+kube_apiserver_enable_admission_plugins:
+  - "PodSecurityPolicy"
+  - "NamespaceLifecycle"
+  - "LimitRanger"
+  - "ServiceAccount"
+  - "DefaultStorageClass"
+  - "DefaultTolerationSeconds"
+  - "MutatingAdmissionWebhook"
+  - "ValidatingAdmissionWebhook"
+  - "ResourceQuota"
+  - "NodeRestriction"
+
+kube_kubeadm_apiserver_extra_args:
+  delete-collection-workers: "3"
```

Comment:
Anyone knows the reason for this value?
This feels like a value that we've kept all the way from when we were using rke https://rancher.com/docs/rke/latest/en/example-yamls/

Reply:
Most probably, I just copied whatever that we're using from ck8s-cluster repo.
But I can remove it.

Reply:
I'm not sure if we have to remove it.
> Number of workers spawned for DeleteCollection call. These are used to speed up namespace cleanup.

It sounds useful to have more than 1.
I'm fine with keeping it the way it is right now.

Reply:
I tried to see if I could find any discussion about this, but I can't find anything other than its 1 as default and rke sets this to 3.
So I guess there's no big pros/cons for this. So lets leave it for now